examples/webgl-vector-layer.js

```diff
@@ -42,3 +43,20 @@ const map = new Map({
     zoom: 1,
   }),
 });
+
+map.on('pointermove', function (evt) {
+  if (evt.dragging) {
+    return;
+  }
+  map.forEachFeatureAtPixel(
+    evt.pixel,
+    function (feature) {
+      window.console.log(feature);
```

Comment:
It would be nice to have this update the page with some detail about the feature (like the [Canvas vector layer example](https://openlayers.org/en/latest/examples/vector-layer.html)) rather than logging.